decimal64.go

```diff
@@ -483,17 +463,17 @@ func size(n uint64) int {
 	return numDigits + 1
 }
 
-// propogateNan returns true if flavor1 should be propogated else false if flavor2 should be propogated
-func propogateNan(flavour1, flavour2 flavor) bool {
-	if flavour1 == flSNaN {
+// propagateNan returns true if flavor1 should be propagated else false if flavor2 should be propagated
+func propagateNan(dp, ep *decParts) *Decimal64 {
+	if dp.fl == flSNaN {
```

Comment:
```go
switch dp.fl {
```

Reply:
``` switch dp.fl{

... 

switch ep.fl{

```

don't see how this is going to make it any more clear

Reply:
changed anyway